src/common/utxobased/db/makeProcessor.ts

```diff
@@ -247,6 +248,7 @@ export async function makeProcessor(
 
     // If there is path info on the address to save but not stored in
     // the previously existing data, save to the by path database.
+    // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
```

Comment:
check 'addressData?.path != null'

Reply:
This is weird either way, if I read this right, we only reach this far if addressData is indeed null.

Reply:
This is weird either way, if I am reading it right, addressData is anyway null.